contracts/GSN/README.adoc

```diff
@@ -4,7 +4,7 @@ _Available since v2.4.0._
 
 This set of contracts provide all the tools required to make a contract callable via the https://gsn.openzeppelin.com[Gas Station Network].
 
-TIP: If you're new to the GSN, head over to our xref:openzeppelin::gsn/what-is-the-gsn.adoc[overview of the system] and basic guide to xref:ROOT:gsn.adoc[creating a GSN-capable contract].
+TIP: If you're new to the GSN, head over to our xref:learn/sending-gasless-transactions.adoc[overview of the system] and basic guide to xref:ROOT:gsn.adoc[creating a GSN-capable contract].
```

Comment:
solidity-docgen shouldn't make a difference here, but I believe this is the correct link: 

```suggestion
TIP: If you're new to the GSN, head over to our xref:learn::sending-gasless-transactions.adoc[overview of the system] and basic guide to xref:ROOT:gsn.adoc[creating a GSN-capable contract].
```

Reply:
I don't think it is, see this link here: https://github.com/OpenZeppelin/openzeppelin-contracts/blame/master/docs/modules/ROOT/pages/gsn.adoc#L5

I think the slash is only used when there are directories inside a component.

Reply:
I don't understand. Don't we want to make a link to the `sending-gassless-transactions` article inside the `learn` component?

Reply:
Corect. See https://docs.antora.org/antora/2.2/asciidoc/page-to-page-xref/

Reply:
Agreed offline that I'm right hehe.